code/game/say.dm

```diff
@@ -106,27 +112,49 @@ var/list/freqtospan = list(
 		return returntext
 	return "[copytext("[freq]", 1, 4)].[copytext("[freq]", 4, 5)]"
 
+/proc/attach_spans(input, list/spans = list())
+	return "[message_spans_start(spans)][input][message_spans_end(spans.len)]"
+
+/proc/message_spans_start(list/spans = list())
+	var/output = ""
+	for(var/S in spans)
```

Comment:
You don't need a list of spans. You just need a list of classes to use as you can combine them like so:

```HTML
<span class="notice warning">Blah</span>
```